__tests__/app.test.js

```diff
@@ -176,3 +176,54 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("Responds with the posted comment", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        username: "icellusedkars",
+        body: "Even a broken clock is right twice a day.",
+      })
+      .expect(201)
+      .then((response) => {
+        console.log(response.body, "<<< comments response");
+        expect(response.body).toEqual({
+          comment: "Even a broken clock is right twice a day.",
+        });
```

Comment:
I would check the other properties here, check that the whole comment has been returned with a comment_id (making the distinction between what's been sent and what's come back) and it should have an article_id attached to it- you can prove you have posted to the correct article from the URL